docs/HACKING.md

```diff
@@ -40,10 +48,16 @@ hack/olm.sh cleanup -c "<OPERATOR_IMAGE>"
 ```
 
 ### Running e2e tests on a cluster
-We need to set up all the environment variables required in [Development workflow](#development-workflow) as well as: 
+The following enviroment variables must be set:
 ```shell script
+# SSH key to be used for creation of cloud-private-key secret
+export KUBE_SSH_KEY_PATH=<path to RSA type ssh key>
 export OPERATOR_IMAGE=<registry url for remote WMCO image>
 export OPENSHIFT_CI=false
+# This variable needs to line up with values in the Makefile and bundle/windows-machine-config-operator.package.yaml
```

Comment:
What values? Please be specific. Sorry about being pedantic but we are going to forget all this in a few weeks and will be struggling to bump a version when it comes time to release.